Index: .idea/dataSources.local.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"dataSourceStorageLocal\" created-in=\"IU-242.20224.300\">\r\n    <data-source name=\"mongodb+srv://user:&lt;db_password&gt;@cluster0.wox2kxx.mongodb.net/?retryWrites=true&amp;w=majority&amp;appName=Cluster0\" uuid=\"1af759d8-42a6-4637-bf24-5743c556dff0\">\r\n      <database-info product=\"Mongo DB\" version=\"8.0.9\" jdbc-version=\"4.2\" driver-name=\"MongoDB JDBC Driver\" driver-version=\"1.20\" dbms=\"MONGO\" exact-version=\"8.0.9\" exact-driver-version=\"1.20\" />\r\n      <case-sensitivity plain-identifiers=\"mixed\" quoted-identifiers=\"mixed\" />\r\n      <user-name>user</user-name>\r\n      <schema-mapping>\r\n        <introspection-scope>\r\n          <node kind=\"schema\" qname=\"@\" />\r\n        </introspection-scope>\r\n      </schema-mapping>\r\n    </data-source>\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/dataSources.local.xml b/.idea/dataSources.local.xml
--- a/.idea/dataSources.local.xml	(revision 0a3e49f87d4dd7b2300afa65184d52e7e1931d57)
+++ b/.idea/dataSources.local.xml	(date 1747137430432)
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="dataSourceStorageLocal" created-in="IU-242.20224.300">
+  <component name="dataSourceStorageLocal" created-in="IU-241.17011.79">
     <data-source name="mongodb+srv://user:&lt;db_password&gt;@cluster0.wox2kxx.mongodb.net/?retryWrites=true&amp;w=majority&amp;appName=Cluster0" uuid="1af759d8-42a6-4637-bf24-5743c556dff0">
       <database-info product="Mongo DB" version="8.0.9" jdbc-version="4.2" driver-name="MongoDB JDBC Driver" driver-version="1.20" dbms="MONGO" exact-version="8.0.9" exact-driver-version="1.20" />
       <case-sensitivity plain-identifiers="mixed" quoted-identifiers="mixed" />
