Index: .idea/dataSources.local.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"dataSourceStorageLocal\" created-in=\"IU-243.21565.193\">\r\n    <data-source name=\"localZero@cluster0.wox2kxx.mongodb.net\" uuid=\"92d3a38c-bf38-4c92-8796-843149de565b\">\r\n      <database-info product=\"Mongo DB\" version=\"8.0.9\" jdbc-version=\"4.2\" driver-name=\"MongoDB JDBC Driver\" driver-version=\"1.20\" dbms=\"MONGO\" exact-version=\"8.0.9\" exact-driver-version=\"1.20\" />\r\n      <case-sensitivity plain-identifiers=\"mixed\" quoted-identifiers=\"mixed\" />\r\n      <user-name>user</user-name>\r\n      <schema-mapping />\r\n    </data-source>\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/dataSources.local.xml b/.idea/dataSources.local.xml
--- a/.idea/dataSources.local.xml	(revision 9711a9d801197271f4cad8c70f6b8f61ea8fe3d4)
+++ b/.idea/dataSources.local.xml	(date 1747915387723)
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="dataSourceStorageLocal" created-in="IU-243.21565.193">
+  <component name="dataSourceStorageLocal" created-in="IU-241.17011.79">
     <data-source name="localZero@cluster0.wox2kxx.mongodb.net" uuid="92d3a38c-bf38-4c92-8796-843149de565b">
       <database-info product="Mongo DB" version="8.0.9" jdbc-version="4.2" driver-name="MongoDB JDBC Driver" driver-version="1.20" dbms="MONGO" exact-version="8.0.9" exact-driver-version="1.20" />
       <case-sensitivity plain-identifiers="mixed" quoted-identifiers="mixed" />
